src/git-sync

```diff
@@ -26,7 +29,11 @@ function preconditions {
   if [ "$1" = "--all" ]; then
     sync_all=true
   elif [ "$(is_feature_branch "$INITIAL_BRANCH_NAME")" = false ]; then
+    # syncing a non-feature branch
```

Comment:
I don't think these comments are necessary.  If the expression in the line above not clear enough?